app/src/main/java/org/jellyfin/androidtv/util/sdk/SdkPlaybackHelper.kt

```diff
@@ -212,6 +212,18 @@ class SdkPlaybackHelper(
 			}
 		}
 
+		BaseItemKind.COLLECTION_FOLDER -> {
```

Comment:
I think adding COLLECTION_FOLDER to the `BaseItemKind.SERIES, BaseItemKind.SEASON, BaseItemKind.BOX_SET, BaseItemKind.FOLDER` line makes more sense. That results in (mostly) the same request.